exporter/metric/metric.go

```diff
@@ -140,83 +139,42 @@ func newMetricExporter(o *options) (*metricExporter, error) {
 
 	cache := map[key]*googlemetricpb.MetricDescriptor{}
 	e := &metricExporter{
-		o:         o,
-		mdCache:   cache,
-		client:    client,
-		startTime: time.Now(),
+		o:       o,
+		mdCache: cache,
+		client:  client,
 	}
 	return e, nil
 }
 
-// InstallNewPipeline instantiates a NewExportPipeline and registers it globally.
-func InstallNewPipeline(opts []Option, popts ...controller.Option) (*controller.Controller, error) {
```

Comment:
I moved this to cloudmonitoring.go, and deprecated it, since that is what upstream otel-go did with their equivalent functions.